src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -4705,10 +4790,27 @@ class Program
                 }
```

Comment:
>} [](http://example.com/codeflow?start=16&length=1)

Consider updating the test to execute these methods, and perhaps additional methods with values, similar to `TypeParameter_01`:
```csharp
static T Create3<T, U>(U x, U y) where T : struct, I<U> => [x, y];
static T? Create4<T, U>(U x, U y) where T : struct, I<U> => [x, y];
```